mongodbatlas/framework/conversion/string.go

```diff
@@ -18,11 +18,21 @@ func TypesListToString(ctx context.Context, set types.List) []string {
 	return results
 }
 
+// IsStringPresent returns true if the string is non-empty.
+func IsStringPresent(strPtr *string) bool {
```

Comment:
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
thanks, moving it in a new commit

Reply:
taking into account what Gus referenced, we could remove the util package altogether and place the type_conversion file within the mongodbatlas package (where it is being used).

Reply:
@gssbzn thanks, you're right about that but at the moment we have very few functions so it would also be a premature optimization to create a package for only one function. as we progress we'll probably extract those from util to a more meaningful package.

Reply:
agree on premature abstraction but the idea is more on preventing a util package, once it is there is really hard to get rid of it so up to you folks on how to resolve

Reply:
thanks @gssbzn , following your advice we see that it doesn't make sense to have a util package only used for the TF repo so we're integrating it, no util package needed.   We'll do it in a new PR as this is getting big and it's not related to this PR specifically